pages/expense.js

```diff
@@ -372,7 +377,16 @@ class ExpensePage extends React.Component {
                     <FormattedMessage id="Edit" defaultMessage="Edit" />
                   </StyledButton>
                 </Box>
-                <ExpenseInfoSidebar isLoading={data.loading} collective={collective} host={host} />
+                <ExpenseInfoSidebar
+                  isLoading={data.loading}
+                  collective={collective}
+                  host={host}
+                  expense={editedExpense || expense}
```

Comment:
**[:sparkles: **Enhancement**]** I would do:

```suggestion
                  expense={status === PAGE_STATUS.VIEW ? expense : editedExpense}
```

It's very minor, but the reason is that `editedExpense` can get out of sync. `expense` is refreshed automatically from the caches, and thanks to the `pollInterval`.